tests/unit/mixins/dirty-confirm-route-test.js

```diff
@@ -142,3 +142,26 @@ test('model is array and checkModelForDirty returns ROLLBACK, checkModelForDirty
   assert.ok(args[3]);
   assert.ok(result);
 });
+
+test('Rollback Dirty Model', function(assert) {
+  var name = 'George',
+    fullName = 'George Castanza';
+
+  var model = Ember.Object.create({
+    name: name,
+    isDirty: false,
+    isSaving: false,
+    rollback: function() {
+      this.name = name;
+    }
+  });
+
+  model.set('name', fullName);
+  model.set('isDirty', true);
+  model.set('isSaving', false);
+
+  assert.equal(model.get('name'), fullName);
```

Comment:
This test doesn't need a concept of name, just set up the model with enough info to call rollback: isDirty true and isSaving false.